### DISCUSSION TITLE
Create a new symfony project in an existing new git repository

### DISCUSSION THREAD

**Discussion by adam314315** [symfony/symfony#49551] [Q&A] [answered]
Hello,

When trying to create a new symfony project in a newly created git repository folder I got the error message Project directory is not empty.

Example

`symfony new myApplication`

With error message

`Project directory myApplication is not empty`

Does anyone find a workaround or cleaner way to do it else than creating the symfony project in a temporary folder and then copying back to the git repository folder ?

**Comment by MatTheCat** [comment, marked as answer]
Hello,

> creating the symfony project in a temporary folder and then copying back to the git repository folder

This seems like the safest thing to do; that way you’re the only one responsible for merging the two folders.

For reference, this issue already got discussed in composer’s repository: https://github.com/composer/composer/issues/1135
